frontend/src/services/typeCheck.js

```diff
@@ -0,0 +1,70 @@
+export const checkVisaType = visaType => {
+    return typeof visaType === "string" && "FBOHL".includes(visaType);
+};
+
+export const checkVisaTypeLst = visaTypeLst => visaTypeLst.every(vt => checkVisaType(vt));
+
+export const checkEmbassyCode = embssyCode => {
```

Comment:
Just put it here to remind myself remember to look into redux subscription for code lst

Reply:
Seems like a bigger refactoring then I previously estimated is needed to fix this issue. We can just import the redux store in this file as it will cause a dependency cycle